CompaniesApp/views.py

```diff
@@ -60,7 +63,7 @@ def joinCompany(request):
 
         return redirect_with_param(request, "company", in_company.name, 'Success! Joined company: '+in_company.name)
     return render(request, 'autherror.html')
-    
+
 def unjoinCompany(request):
```

Comment:
Company model is not indexed using it's members -- search features will have no benefit by knowing which members are part of which company, and if want to find the members of a company we can do a simple query call. So we don't need to update the watson index for it. 

To know more about watson.update_index()
(https://github.com/etianen/django-watson/wiki/registering-models -- scroll down to the middle)